src/component/Button.js

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name }) => (
-  <button type="button" className="btn btn-secondary">
+const Button = ({ name, color, wide }) => (
+  <button type="button" className={`btn orangeColor ${color} ${wide}`}>
```

Comment:
> `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png))

Can we use wide as a boolean instead? The `Bpanel` component seems not to be using this prop at all:
https://github.com/Forison/calculator/blob/eabde4bfc0bb8dde31d47e351d61d5929a3a4db7/src/component/Bpanel.js#L23-L25